corehq/apps/ota/views.py

```diff
@@ -92,15 +95,44 @@ def search(request, domain):
         _soft_assert(False, "Someone in domain: {} tried accessing case search without a config".format(domain), e)
         config = CaseSearchConfig(domain=domain)
 
+    query_addition_id = criteria.pop(SEARCH_QUERY_ADDITION_KEY, None)
+
     fuzzies = config.config.get_fuzzy_properties_for_case_type(case_type)
     for key, value in criteria.items():
         search_es = search_es.case_property_query(key, value, fuzzy=(key in fuzzies))
-    results = search_es.values()
+
+    query_addition_debug_details = {}
+    _add_case_search_addition(request, search_es, query_addition_id, query_addition_debug_details)
+
+    try:
+        results = search_es.values()
+    except Exception as e:
+        notify_exception(request, e.message, details=dict(
+            exception_type=type(e),
+            **query_addition_debug_details
+        ))
+        raise
```

Comment:
should this return a `500` but not just raise the exception? I think as-is we will get two emails every time this happens with the `notify_exception` in place.